importer/helpers/dagbuilder.go

```diff
@@ -30,24 +19,15 @@ type DagBuilderParams struct {
 
 	// DAGService to write blocks to (required)
 	Dagserv dag.DAGService
-
-	// Callback for each block added
-	NodeCB NodeCB
```

Comment:
It's a good idea, and if we come across a use for it sometime down the road we can easily reimplement, but i'd rather not have vestigial 'dead' code laying around doing nothing in the meantime